docs/dev_guide.md

```diff
@@ -0,0 +1,60 @@
+# JupyterLab Developer Guide
+
+## Accessing JupyterLab Development Environment
+
+### 1. Create SSH Tunnel:
+
+Execute the following command to create an SSH tunnel to the remote server (`login1.berkeley.kbase.us`):
+
+```bash
+ssh -f -N -L localhost:44041:10.58.2.201:4041 <ac.anl_username>@login1.berkeley.kbase.us
+```
+   
+### 2. Access JupyterLab Notebooks:
+   
+Open a web browser and navigate to the following URL:
+
+```
+http://localhost:44041
+```
+This will open the JupyterLab Notebook interface running on the remote server.
+
+
+## Accessing MinIO
+Please refer to the [MinIO Guide](minio_guide.md) for instructions on accessing MinIO.
+
+### Read/Write MinIO username and password
+Get the MinIO username and password with read/write permission from the above development JupiterLab environment.
+```python
+import os
+minio_username, minio_password = os.environ['MINIO_ACCESS_KEY'], os.environ['MINIO_SECRET_KEY']
+print(f"MinIO username: {minio_username},\nMinIO password: {minio_password}")
+```
+
+### Naming conventions
+Please adhere to the following naming conventions for MinIO buckets and objects:
+
+#### Source Files:
+Source files are the raw data files that are uploaded to MinIO.
+* Bucket name: `namespace_name`-source
+* File name: The file name should either clearly represent the table name or be formatted in a way that allows a 
+program to easily extract the table name from it.
+
+#### Delta Table Files:
+Delta table files are Parquet files generated by Spark during the creation of a table.
+* Bucket name: `namespace_name`-delta
+
+## Loading Notebooks
+Please create a corresponding loading notebook for each namespace in the `data-loading-notebooks` directory.
+
+Please use the existing loading notebooks as examples.
+
+🚨 **Please DO NOT rerun the loading notebooks in the development environment. Instead, create a new notebook for each 
+new namespace and manually verify the data loading process.**
```

Comment:
I'm not totally sure what this means. If you need to reload the data shouldn't you reuse the notebook?

Reply:
Updated the note to make it more clear. I was trying to prevent people running existing loading notebook and accidentally update existing tables.

Reply:
Is there anything that should be added about ensuring the notebook works when overwriting as opposed to loading clean? I'm not sure there's a guarantee that a notebook can be run without dropping the tables first

Reply:
I think that's fine for now. I haven't thoroughly tested the process of reloading or altering tables yet. For now, I just want to ensure people only load/update their own tables and avoid modifying others' tables. I've observed that people often rerun notebooks to check the output for convenience.